lib/fizzy/execute.cpp

```diff
@@ -141,7 +198,8 @@ inline uint64_t popcnt64(uint64_t value) noexcept
 }
 }  // namespace
 
-Instance instantiate(const Module& module, std::vector<ImportedFunction> imported_functions)
+Instance instantiate(const Module& module, std::vector<ImportedFunction> imported_functions,
+    std::vector<ImportedGlobal> imported_globals)
```

Comment:
```suggestion
    std::vector<ImportedGlobal> imported_globals = {})
```

Reply:
Yeah I didn't do it on purpose, because I'm not a fan of default arguments in general, because they implicitly pass some values... But I agree here it gets very annoying in tests. But I should first add default to `imported_functions`, too, in a separate PR.